pkg/internal/testhelpers/containers.go

```diff
@@ -91,14 +103,22 @@ func (e *ExtensionState) UsePG12() {
 	*e |= postgres12Bit
 }
 
-func (e ExtensionState) UsesTimescaleDB() bool {
+func (e ExtensionState) UsesTimescale1() bool {
```

Comment:
This name is misleading because both ts 1 and ts 2 have this bit set (as well as ts nightly). This really say that we use some version of timescale